### PR TITLE
Telemetry improvements

### PR DESCRIPTION
**Pull request by disq**
- [x] Instrument ConsoleClient
- [x] Collect fetch statistics / resource counts / partial errors
- [ ] Log policy names? Only if cq policies
- [ ] ...

### CODE REVIEW

**Comment by disq** [comment]
I would not merge this before tagging/releasing telemetry 1.0

**Review by roneli — CHANGES_REQUESTED** [review]
Some review comments, would also add to error telemetry the severity/type.

File: internal/telemetry/error.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	//  TODO for fetch get table name / error type
+
+	if anon {
+		span.RecordError(fmt.Errorf("error"), opts...)
+		span.SetStatus(codes.Error, "error")
+		return
+	}
+
+	span.RecordError(err, opts...)
+	span.SetStatus(codes.Error, err.Error())
```
[the remark this answers is not in the document]
Comment (disq) [reply internal/telemetry/error.go:30]: Inverting the if would look like:
`if d, ok := span.(debugModer); !ok || !d.DebugMode() {`
confusing don't you think?

**Review by roneli — APPROVED** [review]
Perfect change imho

**Pull request merged**